lizard_auth_server/forms.py

```diff
@@ -10,12 +10,16 @@
 from django.utils.translation import ugettext_lazy as _
 from itsdangerous import BadSignature
 from itsdangerous import URLSafeTimedSerializer
-
-from lizard_auth_server.models import BILLING_ROLE
-from lizard_auth_server.models import Organisation
-from lizard_auth_server.models import Portal
-from lizard_auth_server.models import THREEDI_PORTAL
-from lizard_auth_server.models import UserProfile
+import jwt
+
+from lizard_auth_server.models import (
```

Comment:
Bah bah bah. Alles op losse regel, dan is het tenminste (onderling) sorteerbaar.